package.json

```diff
@@ -12,7 +12,7 @@
         "README.md"
     ],
     "engines": {
-        "node": ">= 8"
```

Comment:
8 to 20 feels like a big jump. Is the intent to use features that require Node >= 20 in future commits?